src/snowflake/snowpark/mock/_udf_utils.py

```diff
@@ -11,6 +13,33 @@
 )
 from snowflake.snowpark.types import NullType, _NumericType
 
+VARIANT_INPUT_MAPPING = {
+    bytes: lambda x: x.decode("utf-8"),
```

Comment:
🫨 bytes are automatically decoded into str when passed as a variant argument into UDFs?

Reply:
That's what it looks like [here](https://github.com/snowflakedb/snowpark-python/blob/main/tests/integ/test_udf.py#L1246).